cli/azd/cmd/deploy.go

```diff
@@ -64,6 +65,12 @@ func (d *deployFlags) bindCommon(local *pflag.FlagSet, global *internal.GlobalCo
 		false,
 		"Deploys all services that are listed in "+azdcontext.ProjectFileName,
 	)
+	local.StringVar(
+		&d.fromPackage,
+		"from-package",
```

Comment:
@savannahostrowski Thoughts on this flag? `from-package` or `from-packaged`?

Reply:
I probably prefer `from-package`. That reads a bit nicer to me.

I guess I could see someone maaaaybe being like "what package"?, but then again you have to intentionally use this flag so that's probably me being a bit paranoid.